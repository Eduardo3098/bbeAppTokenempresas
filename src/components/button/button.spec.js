import React from 'react';
import {shallow} from 'enzyme';
import Button from './button';

describe('Button component tests', () => {
  test('Should render correctly with primary color', () => {
    const snapshot = shallow(<Button title="TITLE" onPress={jest.fn()} />);
    expect(snapshot).toMatchSnapshot();
  });
  test('Should render correctly with primary color loading', () => {
    const snapshot = shallow(
      <Button status="WAIT" title="TITLE" onPress={jest.fn()} />,
    );
    expect(snapshot).toMatchSnapshot();
  });
});
