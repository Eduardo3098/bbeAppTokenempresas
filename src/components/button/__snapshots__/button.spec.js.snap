// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component tests Should render correctly with primary color 1`] = `
<Styled(Component)
  accessibilityRole="button"
  color="primary"
  disabled={false}
  onPress={[MockFunction]}
>
  <Styled(Text)
    disabled={false}
  >
    TITLE
  </Styled(Text)>
</Styled(Component)>
`;

exports[`Button component tests Should render correctly with primary color loading 1`] = `
<Styled(Component)
  accessibilityRole="button"
  color="primary"
  disabled={false}
  onPress={[MockFunction]}
  status="WAIT"
>
  <Styled(Text)
    disabled={false}
  >
    TITLE
  </Styled(Text)>
</Styled(Component)>
`;
